release-notes/azure/v11.2.1.md

```diff
@@ -1 +1,53 @@
 ## :zap: Giant Swarm Release 11.2.1 for Azure is now active for you! :zap:
+
+**With this release, horizontal pod autoscaling (HPA) is enabled by default for NGINX Ingress Controller (NGINX IC) on selected cluster profiles.**
+
+Here, you will find the NGINX IC to be more elastic thus driving down your costs.
+
+Version development included work with the upstream maintainer. Prior to release, we ran trials with customers. We performed these tests on small and large clusters alike, in order to ensure stability and scalability.
+
+This version also includes an upgrade to the latest upstream NGINX IC 0.30.0, as well as improvements to other components (cluster-operator, kube-state-metrics, chart-operator) as detailed in the changelog.
```

Comment:
@sslavic can you pick top 3 most important component changes for Azure? Seems azure-operator and k8s upgrade might be more important than cluster-operator, chart-operator, etc.?

Reply:
Done, PTAL